### PR TITLE
ENG-12837

### PR DESCRIPTION
**Pull request by akhanzode**
Fraud Detection Application. [ Review Only ]

### CODE REVIEW

File: examples/fraud-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# VoltDB Example App: Fraud Detection (Realtime and In Transaction)
+
+Use Case
+--------
+This application demonstrates how stream of data can be ingested into VoltDB and while ingesting real time decisions such as fraud Detection can be made by simply using power of SQL.
```
Comment (nshi) [inline examples/fraud-detection/README.md]: "fraud prevention"

File: examples/fraud-detection/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+This application performs ingestion of metro card swipes and train activity from 2 different kafka topics. The ingestion from kafka is tied to java stored procedures which detect anamolies and
+computes various VIEWS on data such as
+
+1. Busiest station
+2. Average Wait time for passengers per station
+3. Acceptance rate (Fraud Detection)
```
Comment (nshi) [inline examples/fraud-detection/README.md]: fraud prevention

File: examples/fraud-detection/procedures/frauddetection/CardSwipe.java [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+        final VoltTable[] results = voltExecuteSQL();
+        final VoltTable cardHistoryAtStationisTable = results[0];
+        final VoltTable cardEntriesTable = results[1];
+
+        while (cardHistoryAtStationisTable.advanceRow()) {
+            final byte activity_code = (byte) cardHistoryAtStationisTable.getLong("activity_code");
+            final long stations = cardHistoryAtStationisTable.getLong("stations");
+
+            if (activity_code == ACTIVITY_ENTER) {
+                // Is more than 10 entries at different stations in past hour?
```
Comment (nshi) [inline examples/fraud-detection/procedures/frauddetection/CardSwipe.java]: The comment is a bit out of date, should say 5.

File: examples/fraud-detection/procedures/frauddetection/CardSwipe.java [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+                }
+            }
+        }
+
+        // Is more than 10 consecutive entries in past hour?
+        if (entranceCount >= 10) {
+            return true;
+        }
+
+        // TODO: 50% more entries than average at station with small number of distinct card?
```
Comment (nshi) [inline examples/fraud-detection/procedures/frauddetection/CardSwipe.java]: We can probably remove this todo.

File: examples/fraud-detection/ddl.sql [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+  CONSTRAINT PK_stations PRIMARY KEY (station_id)
+);
+
+CREATE TABLE trains (
+  train_id            SMALLINT          NOT NULL,
+  name                VARCHAR(25) NOT NULL,
+  CONSTRAINT PK_trains PRIMARY KEY (train_id)
+);
+
+CREATE TABLE wait_time (
```
Comment (xinjiacs) [inline examples/fraud-detection/ddl.sql]: Move table "wait_time" into partitioned table section.

**Pull request merged**